internal/commands/commands.go

```diff
@@ -237,16 +238,38 @@ be modified.`,
 	}
 
 	Function = cli.CommandDefinition{
-		Command:     &function.Command{},
-		Use:         function.CommandUse,
-		Aliases:     function.CommandAliases,
-		Description: "Run a function from your Realm app",
-		Help: `Realm Functions allow you to define and execute server-side logic for your
+		Use:         "function",
+		Aliases:     []string{"functions"},
+		Description: "Manage the functions of your Realm app",
+		SubCommands: []cli.CommandDefinition{
+			{
+				Command:     &function.CommandRun{},
+				Use:         "run",
+				Description: "Run a function from your Realm app",
+				Help: `Realm Functions allow you to define and execute server-side logic for your
 Realm app. Once you select and run a function for your Realm app, the
 following will be displayed:
  - A list of logs, if present
  - The function result as a document
  - A list of error logs, if present
 `,
+			},
+		},
+	}
+
+	Logs = cli.CommandDefinition{
+		Use:         "logs",
+		Aliases:     []string{"log"},
+		Description: "Interact the logs of your Realm app",
```

Comment:
looks like we need a "with" here

Reply:
I missed the other help texts as well for the subcommand